user-guide/running/product-management/products/adding-products/index.md

```diff
@@ -68,7 +68,7 @@ In the **Prices** section define:
 - **Unit of product** - measure of a previously entered value.
 - **Reference amount** - the base amount.
 - **Reference unit** - measure of a previously entered value.
-- **Discounts**. Learn how to set up discounts [here](xref:/user-guide/marketing/promotional/discounts/index.html).
```

Comment:
Apparently you did not download the latest changes from the "master" branch. This has caused some changes made by others to be erased in your pull request.

Now there are no problems with missing images when assembling the documentation. However, you need to have in your request for retraction was only the changes you have made. I added a step-by-step description of how to do this correctly in the [CONTRIBUTION.md](https://github.com/nopSolutions/nopCommerce-Docs/blob/master/CONTRIBUTING.md#forking-and-submitting-pull-requests) file. Just follow the instructions and everything should work out.

Reply:
Hi @DmitriyKulagin,
Thank you for the document link. I updated my fork before commit but I think sync part was missing.
Today I'll check again.
Thanks